draft-ietf-acme-acme.md

```diff
@@ -790,7 +806,7 @@ externalAccountRequired (optional, boolean):
 new-account requests include an "externalAccountBinding" field associating the
 new account with an external account.
 
-Clients access the directory by sending a GET request to the directory URL.
+Clients access the directory by sending a POST-as-GET request to the directory URL.
```

Comment:
This should be `GET`, as this is about the directory (which is explicitly mentioned as `GET` above).

Reply:
Yep.  Overzealous search-and-replace :)